Tests/js/views/dashboard/assets/ez-dashboardblockallcontentview-tests.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentview-tests', function (Y) {
+    'use strict';
+
+    var renderTest,
+        getDataTest,
+        uiEventsTest,
+        CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_CONTENT = '.ez-allcontent-block-content',
+        SELECTOR_ROW = '.ez-allcontent-block-row',
+        SELECTOR_OUTSIDE = '.outside',
+        PATH_STRING = 'imagine-dragons',
+        ROOT_LOCATION_ID = '23',
+        VIEW_CONFIG = {container: '.container'};
+
+    renderTest = new Y.Test.Case({
+        name: 'eZ Dashboard All Content Block View render test',
+
+        setUp: function () {
+            this.rootLocation = new Y.Model({
+                locationId: ROOT_LOCATION_ID,
+                pathString: PATH_STRING
+            });
+            this.view = new Y.eZ.DashboardBlockAllContentView(Y.merge(VIEW_CONFIG, {rootLocation: this.rootLocation}));
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        'Should render view in the loading state': function () {
+            var view = this.view,
+                templateCalled = false,
+                origTpl = view.template,
+                container = view.get('container');
+
+            view.template = function () {
+                templateCalled = true;
+
+                return origTpl.apply(this, arguments);
+            };
+
+            view.render();
+
+            Y.Assert.isTrue(templateCalled, 'The template should have been used to render view');
+            Y.Assert.areSame(0, container.one(SELECTOR_CONTENT).all('tr').size(), 'Should not render any row');
+            Y.Assert.isTrue(container.hasClass(CLASS_LOADING), 'Should add the loading state CSS class to the view container');
+        }
+    });
+
+    getDataTest = new Y.Test.Case({
```

Comment:
there's no *getData*, I would call it `searchEventTest` and fix the same way any *get data* usage in this test case.